bundles/CreateProject/Form/CreateProjectForm.php

```diff
@@ -97,8 +97,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'attr' => ['class' => 'form-control'],
                 'help_attr' => ['class' => 'form-text text-muted'],
                 'help' => 'create_project_form.team.help',
-            ])
-            ->add('account', ChoiceType::class, [
+            ]);
+        if ($options['user_permissions']->permissions->ADMINISTER->havePermission) {
```

Comment:
This does not looks right. I suppose it works, but it may make sense to put this logic into a `hasPermission` helper function in this class.